architecture/index.adoc

```diff
@@ -0,0 +1,70 @@
+{product-title} assigns different roles to hosts. These roles define the function of a machine in a cluster.
+The cluster contains definitions for the standard control plane and worker role types.
+
+You can use Operators to package, deploy, and manage services on the control plane. With Operators, you can use `kubectl` and `oc` commands.
+Operators are important components in {product-title} because they provide the following services:
+
+* Perform health checks
+* Enable the way to watch an application
+* Manage over-the-air updates
+* Ensure applications stay in the specified state
+
+[id="about-containerized-applications-for-developers"]
+== About containerized applications for developers
+
+As a developer, you can use different tools, methods, and formats to xref:../architecture/understanding-development.adoc#understanding-development[develop your containerized application] based on your unique requirements, for example:
+
+- Use various build-tool, base-image, and registry options to build a simple container application.
+- Use supporting components such as OperatorHub and templates to develop your application.
+- Package and deploy your application as an Operator.
+
+You can also create a Kubernetes manifest and store it in a Git repository.
+Kubernetes works on basic units called pods. A pod is a single instance of a running process in your cluster. Pods can contain one or more containers.
+You can create a service by grouping a set of pods and their access policies.
+Services provide permanent internal IP addresses and host names for other applications to use as pods are created and destroyed. Kubernetes defines workloads based on the type of your application.
+
+[id="coreos-and-ignition"]
+== About {op-system-first} and Ignition
+
+As a cluster administrator, you can perform the following tasks:
+
+** Learn about the next generation of xref:../architecture/architecture-rhcos.adoc#architecture-rhcos[single-purpose container operating system technology].
+** Choose how to configure {op-system-first}
+** Choose how to deploy {op-system-first}:
+*** Installer-provisioned deployment
+*** User-provisioned deployment
+
+The {product-title} installation program creates the Ignition configuration files that you need to deploy your cluster.
+{op-system-first} uses Ignition during the initial configuration to perform common disk tasks, such as partitioning, formatting, writing files, and configuring users.
+During the first boot, Ignition reads its configuration from the installation media or the location that you specify and applies the configuration to the machines.
+
+You can learn how xref:../architecture/architecture-rhcos.adoc#architecture-rhcos[Ignition works], the process for an {op-system-first} machine in an {product-title} cluster, view Ignition configuration files, and change Ignition configuration after an installation.
```

Comment:
* the process for an -> the process for a 
* I'm assuming that this link for how ignition works is supposed to jump further down, to https://deploy-preview-40012--osdocs.netlify.app/openshift-enterprise/latest/architecture/architecture-rhcos.html#about-ignition_architecture-rhcos

Reply:
I have included a ref to the assembly in which we have the content `How Ignition works` from the module titled **rhcos-about.adoc**